### PR TITLE
Update on readme.md file (Documentation)

### PR DESCRIPTION
**Pull request by AminTorabi-NOAA**
This is just a first draft of an update on readme file for documentation. 

## Additions

1- A folder name `images` under the `doc`  is added to include all the images that will be use in readme.md file
2- Structure of folders added
3- Installation requirement for windows user (on WSL) is added. @JurgenZach-NOAA if you have any additional comments on installation and possible issues on WSL please feel free to add

## Removals

-

## Changes

-

## Testing

1.

## Notes

-

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows project standards (link if applicable)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Target Environment support

- [ x] Windows
- [x ] Linux
- [ ] Browser

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

**Review by JurgenZach-NOAA — DISMISSED** [review]
Good to go

File: readme.md [hunk 1]
```diff
@@ -59,13 +97,50 @@ $ cd test/LowerColorado_TX
 $ python3 -m nwm_routing -V3 -f test_AnA.yaml
```
Comment (kumdonoaa) [inline readme.md:97]: I think we can include two main test cases one for NHD and the other for HYfeature.  So, for example, -V4 -f test_AnA_V4_NHD.yaml and -V -f test_AnA_V4_HYFeature.yaml as V4 is the most update main function.

File: readme.md [hunk 2]
```diff
@@ -2,16 +2,51 @@
...
+T-route, a dynamic channel routing model, offers a comprehensive solution for river network routing problems. It is designed to handle 1-D channel routing challenges in vector-based river network data, such as the USGS's NHDPlus High Resolution dataset, and OGC WaterML 2.0 Surface Hydrology Features (HY_Features) data model used in NWMv3.0.
```
Comment (shorvath-noaa) [inline readme.md:5]: the "t" in "t-route" should not be capitalized.

Comment (shorvath-noaa) [inline readme.md:5]: Is HYFeatures used for NWMv3.0? I don't think it is. Do you mean NextGen framework?

File: readme.md [hunk 3]
```diff
@@ -2,16 +2,51 @@
...
 
-t-route development is rigorously aligned with and guided by the NOAA Office of Water Prediction mission: *Collaboratively research, develop and deliver timely and consistent, state-of-the-science national hydrologic analyses, forecast information, data, guidance and equitable decision-support services to inform essential emergency management and water resources decisions across all time scales.*  
+Expanding its capabilities, t-route now supports the OGC WaterML 2.0 Surface Hydrology Features (HY_Features) data model, facilitating the management of complex acyclic network connectivity. This extends t-route's applicability to simulating reservoirs, assimilating reservoir release data, streamflow nudging, and provides users with a choice of routing solutions, including Muskingum-Cunge, Diffusive Wave, or Dynamic Wave.
```
Comment (shorvath-noaa) [inline readme.md:9]: This reads like reservoir routing and DA are a result of applying t-route on the HYFeatures network, which is not true. Those features exist for the NHDNetwork as well.

File: readme.md (hunk 2 above)
Comment (AminTorabi-NOAA) [reply readme.md:5]: Yes NextGen. I'll change that

File: readme.md (hunk 1 above)
Comment (AminTorabi-NOAA) [reply readme.md:97]: Good point. I'll update that

File: readme.md (hunk 3 above)
Comment (AminTorabi-NOAA) [reply readme.md:9]: Is this better? 
"Expanding its capabilities, t-route now supports the OGC WaterML 2.0 Surface Hydrology Features (HY_Features) data model, facilitating the management of complex acyclic network connectivity. HY_Features data model provides users with a choice of routing solutions, including Muskingum-Cunge, Diffusive Wave, or Dynamic Wave."

**Review by kumdonoaa — APPROVED** [review]

**Pull request merged**